lib/active_resource/base.rb

```diff
@@ -1683,15 +1700,6 @@ def connection(refresh = false)
         self.class.connection(refresh)
       end
 
-      # Update the resource on the remote service.
-      def update
```

Comment:
What calls this method in the framework. Nothing? I think it is `save`. In that case, I'd rename this method like we did in Active Record to `_update`. And then call `_update` in `update` without arguments and deprecate `update` without arguments in favor of calling `save`